kouta-backend/src/main/scala/fi/oph/kouta/repository/toteutusDAO.scala

```diff
@@ -199,6 +212,10 @@ sealed trait ToteutusSQL extends ToteutusExtractors with ToteutusModificationSQL
     sql"""select toteutus_oid, tarjoaja_oid from toteutusten_tarjoajat where toteutus_oid in (#${createOidInParams(oids)})""".as[Tarjoaja]
   }
 
+  def selectToteutuksetByOids(toteutusOids: List[ToteutusOid]) =
+    sql"""#$selectToteutusSql
+          where t.oid in (#${createOidInParams(toteutusOids)})"""
```

Comment:
Voikohan tässä tulla vastaan In -lauseen maksimikoko (= kuinka monta arvoa listalla voi olla).. En osaa suoraan sanoa mikä maksimimäärä Postgre:ssa on, tai onko edes yksiselitteistä maksimimäärää. Oraclessa vastaava oli muistaakseni 250, siitä johtuen täytyi viritellä useita OR:illa erotettuja In -lauseita joissa kussakin max 250 (yök)

Reply:
Yritin guuggeloida tätä asiaa ja näyttäisi, että tuollaista maksimi parametrimäärää ei ole tai se on todella iso. Tää oli paras ketju, mitä löysin aiheesta: https://stackoverflow.com/a/47729784

Reply:
Jeps, ei tuolle varmaan kannata tehdä tässä vaiheessa mitään. Ja vaikea kuvitella että toteutusten määrä olisi missään tilanteessa älyttömän suuri (tyyliin > 10000)